Sources/KSCrashRecording/include/KSCrash.h

```diff
@@ -105,10 +105,24 @@ NS_ASSUME_NONNULL_BEGIN
 
 #pragma mark - API -
 
-/** Init KSCrash instance with custom base path. */
-- (instancetype) initWithBasePath:(NSString*) basePath;
+- (instancetype) init __attribute((unavailable("Use `sharedInstance` instead.")));
++ (instancetype) new __attribute((unavailable("Use `sharedInstance` instead.")));
```

Comment:
Better use `NS_UNAVAILABLE`. Setting it on `init` disables `new` automatically as well.